test/e2e/storage/utils/utils.go

```diff
@@ -818,3 +819,17 @@ func GetSizeRangesIntersection(first e2evolume.SizeRange, second e2evolume.SizeR
 	}
 	return "", fmt.Errorf("intersection of size ranges %+v, %+v is null", first, second)
 }
+
+// GetVolumeAttachment attempts to get the associated VolumeAttachment for the passed in PV.
+// It returns the VolumeAttachment, assuming one is found, and any generated errors
+func GetVolumeAttachment(cs clientset.Interface, pv *v1.PersistentVolume, provisioner, nodeName string) (*storagev1.VolumeAttachment, error) {
```

Comment:
Please let `csi_mock_volume.go` use this function too.

Reply:
This has been updated, with some changes to the `getVolumeHandle` function that was previously in use.